src/derivatives/05-diffusion-derivatives.md

```diff
@@ -381,8 +379,14 @@ another.
 
     -   Utilised basis functions:
 
+        -   `MRtrix3`
+
         ![MRtrix3 SH basis functions](https://latex.codecogs.com/gif.latex?Y_{lm}(\theta,\phi)=\begin{Bmatrix}&space;0&\text{if&space;}l\text{&space;is&space;odd},\\&space;\sqrt{2}\times\text{Im}\left[Y_l^{-m}(\theta,\phi)\right]&\text{if&space;}m<0,\\&space;Y_l^0(\theta,\phi)&\text{if&space;}m=0,\\&space;\sqrt{2}\times\text{Re}\left[Y_l^m(\theta,\phi)\right]&\text{if&space;}m>0\\&space;\end{Bmatrix})
 
+        -   `Descoteaux`
+
+        ![Descoteaux SH basis functions](https://latex.codecogs.com/gif.latex?Y_{lm}(\theta,\phi)=\begin{Bmatrix}&space;0&\text{if&space;}l\text{&space;is&space;odd},\\&space;\sqrt{2}\times\text{Re}\left[Y_l^{-m}(\theta,\phi)\right]&\text{if&space;}m%3C0,\\&space;Y_l^0(\theta,\phi)&\text{if&space;}m=0,\\&space;\sqrt{2}\times\text{Im}\left[Y_l^m(\theta,\phi)\right]&\text{if&space;}m%3E0\\&space;\end{Bmatrix})
```

Comment:
```suggestion
        ![Descoteaux SH basis functions](https://latex.codecogs.com/gif.latex?Y_{lm}(\theta,\phi)=\begin{Bmatrix}&space;0&\text{if&space;}l\text{&space;is&space;odd},\\&space;\sqrt{2}\times\text{Re}\left[Y_l^{-m}(\theta,\phi)\right]&\text{if&space;}m<0,\\&space;Y_l^0(\theta,\phi)&\text{if&space;}m=0,\\&space;\sqrt{2}\times\text{Im}\left[Y_l^m(\theta,\phi)\right]&\text{if&space;}m>0\\&space;\end{Bmatrix})
```

Also: I've not used this basis myself, but I have vague recollection of a discussion many years ago about how this basis involved negation of every second even degree, such that coefficients for a disc-like response function would be all positive (whereas in the *MRtrix3* basis they alternate between positive and negative). Is this missing from here, or am I misinformed or mis-remembering?

Reply:
I don't see that in the code or in the paper, so I don't think so.

Reply:
@jdtournier Any recollection?

Reply:
Nope, sorry... We may have discussed the idea, but there was never to my knowledge any basis in MRtrix that would explicitly invert harmonics like that. 

As far as I know, the only ambiguity when it comes to the sign is the [Condon–Shortley phase term](https://en.wikipedia.org/wiki/Spherical_harmonics#Condon%E2%80%93Shortley_phase), but that only affects the odd *m* terms - it would leave the *m*=0 terms - and hence the response function - unaffected.

Reply:
OK cool, writing it off as a historical misunderstanding on my part.